dev/provider_packages/tag_providers.sh

```diff
@@ -35,6 +35,18 @@ do
     { git tag "${tag}" -m "Release $(date '+%Y-%m-%d') of providers" && tags+=("$tag") ; } || true
    fi
 done
+
 if [[ -n "${tags:-}" && "${#tags}" -gt 0 ]]; then
-   git push $remote "${tags[@]}"
+   if git push $remote "${tags[@]}"; then
+       echo "Tags pushed successfully"
```

Comment:
Normally the script outputs:

```
➜  airflow git:(main) ./dev/provider_packages/tag_providers.sh
Enumerating objects: 82, done.
Counting objects: 100% (82/82), done.
Delta compression using up to 10 threads
Compressing objects: 100% (82/82), done.
Writing objects: 100% (82/82), 3.33 KiB | 682.00 KiB/s, done.
Total 82 (delta 79), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (79/79), done.
To https://github.com/apache/airflow.git
 * [new tag]               providers-airbyte/3.4.0rc1 -> providers-airbyte/3.4.0rc1
 * [new tag]               providers-alibaba/2.6.0rc1 -> providers-alibaba/2.6.0rc1
 * [new tag]               providers-amazon/8.8.0rc1 -> providers-amazon/8.8.0rc1
 * [new tag]               providers-apache-beam/5.3.0rc1 -> providers-apache-beam/5.3.0rc1
 * [new tag]               providers-apache-cassandra/3.3.0rc1 -> providers-apache-cassandra/3.3.0rc1
 * [new tag]               providers-apache-drill/2.5.0rc1 -> providers-apache-drill/2.5.0rc1
 * [new tag]               providers-apache-druid/3.6.0rc1 -> providers-apache-druid/3.6.0rc1
 * [new tag]               providers-apache-flink/1.2.0rc1 -> providers-apache-flink/1.2.0rc1
 * [new tag]               providers-apache-hdfs/4.2.0rc1 -> providers-apache-hdfs/4.2.0rc1
 * [new tag]               providers-apache-hive/6.2.0rc1 -> providers-apache-hive/6.2.0rc1
 * [new tag]               providers-apache-impala/1.2.0rc1 -> providers-apache-impala/1.2.0rc1
 * [new tag]               providers-apache-kafka/1.2.0rc1 -> providers-apache-kafka/1.2.0rc1
 
```
 
 If I read this right now the script will not output this but `Tags pushed successfully` ?

Reply:
I think the original code was correct (and simpler).

```
if git push $remote "${tags[@]}"; then
```

would still output the stdout/stderr to terminal.

Reply:
I understand the point of simpler here and I second it. Would like to run it by @eladkal once too. @eladkal wdyt? I plan to bring back the HEAD to one commit before.

Reply:
Sure if the output is present then lets.
I will test it with next release